Komiser crashing when scanning AWS accounts

**Describe the bug**
I'm seeing Komiser crashing with the following error:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1ecd3a1]
goroutine 67 [running]:
github.com/tailwarden/komiser/providers/aws/ec2.getEC2Relations(0xc01116f8f0, {0xc01291d140, 0x22})
	/home/runner/work/komiser/komiser/providers/aws/ec2/instances.go:202 +0x561
github.com/tailwarden/komiser/providers/aws/ec2.Instances({0x3b758a0, 0x5aa98c0}, {0xc00b3ff7a0, 0x0, {0x0, 0x0}, 0x0, 0x0, 0x0, 0x0, ...})
	/home/runner/work/komiser/komiser/providers/aws/ec2/instances.go:136 +0x111b
github.com/tailwarden/komiser/providers/aws.FetchResources.func1()
	/home/runner/work/komiser/komiser/providers/aws/aws.go:119 +0x99
github.com/tailwarden/komiser/providers.(*WorkerPool).worker(0xc00013b8c0)
	/home/runner/work/komiser/komiser/providers/providers.go:85 +0x3a
created by github.com/tailwarden/komiser/providers.(*WorkerPool).Start in goroutine 28
	/home/runner/work/komiser/komiser/providers/providers.go:69 +0x25
```

**To Reproduce**
Steps to reproduce the behavior:
1. Add multiple AWS accounts to Komiser
2. Wait a couple of minutes

**Expected behavior**
Komiser does not crash

**Screenshots**
If applicable, add screenshots or Loom video to help explain your problem.

**Desktop (please complete the following information):**
 - OS: MacOS Ventura
 - Browser: Firefox
 - Version: Komiser 3.1.5 (Docker container)

--- a/providers/aws/ec2/instances.go
+++ b/providers/aws/ec2/instances.go
@@ -174,9 +174,9 @@ func Instances(ctx context.Context, client providers.ProviderClient) ([]models.R
 }
 
 func getEC2Relations(inst *etype.Instance, resourceArn string) []models.Link {
-	
+
 	var rel []models.Link
-	// Get associated security groups	
+	// Get associated security groups
 	for _, sgrp := range inst.SecurityGroups {
 		rel = append(rel, models.Link{
 			ResourceID: *sgrp.GroupId,
@@ -197,21 +197,25 @@ func getEC2Relations(inst *etype.Instance, resourceArn string) []models.Link {
 		})
 	}
 
-	// Get associated VPC
-	rel = append(rel, models.Link{
-		ResourceID: fmt.Sprintf("%s:vpc/%s", resourceArn, *inst.VpcId),
-		Type:       "VPC",
-		Name:       *inst.VpcId,
-		Relation:   "USES",
-	})
+	if inst.VpcId != nil {
+		// Get associated VPC
+		rel = append(rel, models.Link{
+			ResourceID: fmt.Sprintf("%s:vpc/%s", resourceArn, *inst.VpcId),
+			Type:       "VPC",
+			Name:       *inst.VpcId,
+			Relation:   "USES",
+		})
+	}
 
-	// Get associated Subnet
-	rel = append(rel, models.Link{
-		ResourceID: fmt.Sprintf("%s:subnet/%s", resourceArn, *inst.SubnetId),
-		Name:       *inst.SubnetId,
-		Type:       "Subnet",
-		Relation:   "USES",
-	})
+	if inst.SubnetId != nil {
+		// Get associated Subnet
+		rel = append(rel, models.Link{
+			ResourceID: fmt.Sprintf("%s:subnet/%s", resourceArn, *inst.SubnetId),
+			Name:       *inst.SubnetId,
+			Type:       "Subnet",
+			Relation:   "USES",
+		})
+	}
 
 	// Get associated Keypair
 	if inst.KeyName != nil {

